packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-split.yaml

```diff
@@ -0,0 +1,8 @@
+Resources:
+  Bucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Split [ 
+        ' ', 
+        !ImportValue SomeSharedBucketName
+      ]
```

Comment:
Can you also use short forms like this?

```
      BucketName: !Split
      - ' ', 
      -  !ImportValue SomeSharedBucketName
```

If the answer  is "yes", I would love to have a few tests checking that as well.

Reply:
docs don't show any examples, but I'll test that

Reply:
I can confirm that a bucket name defined like this 

`      BucketName:
        !If
        - AlwaysTrueCond
        - !FindInMap
            - RegionMap
            - region-1
            - HVM64
        - !FindInMap [ RegionMap, region-1, HVMG2 ]`

is successfully deployed by cloudformation.

Reply:
Sweet. Can we add a test for this form as well?

Reply:
Yes. Should we have one for each Cfn function?

Reply:
Nah. Just have it for one of them, like `If`, or `Split`.